docs/understanding-radicle/glossary.md

```diff
@@ -21,162 +23,148 @@ changes committed to a [project][pr] history along with who made them and when.
 [project][pr].
 
 ## display name
-A changeable and non-unique human-readable name chosen by the [user][us]. Like
-a "nickname" for a [Radicle ID][ri]. You can change it in your profile at any
-time.
+A changeable and non-unique human-readable name chosen by the [user][us]. Like a
+"nickname" for a [Radicle ID][ri]. This will be possible to change in the
+future. 
 
 ## follow
-Following a [project][pr] or [user][us] replicates it's data to the
-[user][us]'s machine. This allows a [user][us] to subscribe to the changes of a
-remote [repository][re] and signal interest by further replicating the data
-within their social graphs.
+Following a [project][pr] or [peer][us] replicates it's data to the your machine
+(See [Tracking](/how-it-works.md/#tracking)). This allows the follower to
+subscribe to updates from the project / peer and signal interest by further
+replicating the data across their network, making it available to their social
+graph.
 
 ## git
 A free and open source distributed version control system.
 
 ## gossip
 A means of peer-to-peer communication that relies on the interactions of peers
-to disseminate information. Participants in the Radicle network share and
-spread data by keeping redundant copies of repositories locally and sharing
-updates with peers. This allows data to be disseminated according to use and
-value: the more peers who are interested in certain data the more available a
-[project][pr] is made to the network.
+to disseminate information. Participants in the Radicle network share and spread
+data by keeping redundant copies of proejcts locally and sharing updates with
+peers. This allows data to be disseminated according to use and value: the more
+peers who are interested in certain data the more available a [project][pr] is
+made to the network. See [Gossip](/how-it-works.md/#replication-model)
 
 ## maintainer
-A [user][us] that manages the work of a [project][pr]'s [repository][re] by
-triaging issues and adjusting permissions of other [user][us]s who wish to
-contribute to the [project][pr]. A maintainer is explicitly associated with the
-[project][pr] at the time of creation or later, via the [project][pr] metadata.
+A [user][us] that manages updates to the canonical view of a project. A
+maintainer is explicitly associated with the [project][pr] at the time of
+creation or later, via the [project][pr] metadata [See Data
+Model](/how-it-works.md/#data-model), but there may be multiple maintainers of a
+project, all maintaining their own views. At creation, the creator of the
+[project][pr] is the sole maintainer.
 
 All (or a quorum of) maintainer signatures are required to write to the
-canonical upstream branch of a [project][pr]. Maintainers are a subset of
-contributors. There can be more than one per [project][pr], and at creation,
-the creator of the [project][pr] is the sole maintainer.
+canonical upstream branch of a [project][pr]. 
```

Comment:
I think this description could be improved. There's no canonical branch really.

Reply:
Maybe we just take last sentence out bc it's not relevant yet.